chains/benchmarks/bigbench-binary-classification.aipl

```diff
@@ -0,0 +1,49 @@
+# in: list of models; out: % accuracy in classifying the given task
+
+# make all pairwise combinations of model and task
+!python
+with open('bigbench-tasks.txt') as f:
+    tasks = f.read().split('\n')
+    tasks = [t for t in tasks if '#' not in t]
+with open('models.txt') as f:
+    models = f.read().split('\n')
+    models = [m for m in models if '#' not in m]
+s = 'model,task\n' + "\n".join([model+","+task for model in models for task in tasks])
+with open('model-task.csv', 'w') as f:
+    f.write(s)
+!csv-parse model-task.csv
```

Comment:
This whole block is clever, but we should figure out how a `!cross` or something should work.

Reply:
totally, if you're doing that separately then we can hold this PR off until that's merged

Reply:
should be merged now!